upgrade/1.2/scripts/rebuild/ncn-rebuild-worker-nodes.sh

```diff
@@ -41,7 +41,7 @@ state_recorded=$(is_state_recorded "${state_name}" ${target_ncn})
 if [[ $state_recorded == "0" ]]; then
     echo "====> ${state_name} ..."
 
-    workers="$(kubectl get node --selector='!node-role.kubernetes.io/master' -o name | sed -e 's,^node/,,' | paste -sd,)"
+    workers="$(kubectl get node --selector='!node-role.kubernetes.io/master' -o name | grep -v ${target_ncn} | sed -e 's,^node/,,' | paste -sd,)"
```

Comment:
This kubectl get node seems jank, if all we want is the names why not just this?

```
ncn-m001:~ # kubectl get nodes --selector='!node-role.kubernetes.io/master' --no-headers=true -o custom-columns=NAME:.metadata.name
ncn-w001
ncn-w002
ncn-w003
```

Instead of this output?
```
ncn-m001:~ # kubectl get node --selector='!node-role.kubernetes.io/master' -o name
node/ncn-w001
node/ncn-w002
node/ncn-w003
```

Reply:
Great comment -- fixed..